app/assets/stylesheets/comp-graph-dc.scss

```diff
@@ -64,3 +64,33 @@
     display: inline-block;
   }
 }
+
+//Overlays
+.overlay-container {
```

Comment:
This could be turn into a component (partial) to be used all around

Reply:
Yes, go ahead.